src/ejp_uat.js

```diff
@@ -848,6 +848,7 @@ function ejpUatParseXmlTerms(url, xmlString) {
     for ( var i = 0; i < termNodes.length; i++ ) {
         termNode = termNodes[i];
         aboutLink = termNode.getAttribute('rdf:about');
+        if ( !aboutLink ) continue; // KF - skip if we cant found aboutLink
```

Comment:
Stylistic note, with good version control systems like Git it's a lot less important to annotate these kinds of changes with named comments — the output of tools `git blame` will show per-line authorship and commit info, which can then in turn be traced back to this pull request pretty easily. So these days I generally recommend not to bother with line-by-line change comments, although obviously they don't hurt.

Separately, my personal preference is to avoid single-line conditionals since they can make it easy to miss important pieces of control-flow logic when skimming code. You're being consistent with the existing code here, but as a general matter I think it's better to write similar logic out more verbosely:

```
if (!aboutLink) {
    continue;
}
```